ExtractorUtils.Test/unit/Unstable/RuntimeTest.cs

```diff
@@ -41,7 +41,8 @@ private string GetConnectionConfig()
             return @"
 project: project
 base-url: https://api.cognitedata.com
-integration: test-integration
+integration:
+    external-id: test-integration
```

Comment:
This change is breaking, but I presume it's an experimental API?

Reply:
Yep. Everything under Unstable is, reasonably enough, experimental.